share/man/man4/vt.4

```diff
@@ -358,6 +364,14 @@ To set black and white colors of console palette
 .Pp
 .Dl kern.vt.color.0.rgb="10,10,10"
 .Dl kern.vt.color.15.rgb="#f0f0f0"
+.Pp
+Load the 8x16 font in
+.Xr loader.conf 5
+from
+.Pa /boot/fonts/*.fnt[.gz]
+at boot:
+.Pp
+.Dl screen.font="8x16"
 .Sh SEE ALSO
```

Comment:
please add loader.conf(5) to the SEE ALSO section.

Reply:
Fixed thank you so